geth/wrapper.py

```diff
@@ -101,7 +109,7 @@ def construct_test_chain_kwargs(**overrides: Any) -> dict[str, Any]:
             os.path.join(tempfile.mkdtemp(), "geth.ipc"),
         )
 
-    overrides.setdefault("verbosity", "5")
+    overrides.setdefault("verbosity", 5)
```

Comment:
Same here with my comment about the port... should we use `str` or `int`? Should we maybe be flexible with these? I haven't poked around much but is there a reason to go `int` with one kwarg and `str` with another?

Reply:
No, probably should have gone the other way, making the `GethKwargs` field a `str` instead. More broadly, yes, I think opening some of these up to be `int` or `str` would be an appropriate move.

Reply:
Looking further, it seems like `subprocess.Popen` should only be called with a string or sequence of strings. The docs reference it as "a string or sequence of program arguments" - https://docs.python.org/3/library/subprocess.html#popen-constructor

But in one place it does note:

> The value for args may be a single string or a list of strings, depending on platform.

I would lean then towards changing everything in `geth_kwargs` that is currently an `int` to a `str`, then we don't need to worry about `str`ing things in `construct_popen_command`

Reply:
I think that's the best idea as well 👍🏼